lib/mayaUsd/nodes/proxyShapeBase.cpp

```diff
@@ -479,6 +481,19 @@ MStatus MayaUsdProxyShapeBase::initialize()
     retValue = addAttribute(lockedLayersAttr);
     CHECK_MSTATUS_AND_RETURN_IT(retValue);
 
+    layerManagerAttr = typedAttrFn.create(
+        "layerManager", "lymgr", MFnData::kString, MObject::kNullObj, &retValue);
+    typedAttrFn.setStorable(true);
+    typedAttrFn.setWritable(true);
+    typedAttrFn.setReadable(true);
+    typedAttrFn.setInternal(true);
+    typedAttrFn.setHidden(true);
+    typedAttrFn.setDisconnectBehavior(MFnNumericAttribute::kReset); // on disconnect, reset to Null
+    typedAttrFn.setAffectsAppearance(false);
+    CHECK_MSTATUS_AND_RETURN_IT(retValue);
+    retValue = addAttribute(layerManagerAttr);
+    CHECK_MSTATUS_AND_RETURN_IT(retValue);
+
```

Comment:
The new proxy shape attribute will keep the layer manager node UUID as a string.